### PR TITLE
[Enhancement] Add BE down tolerate time for colcoate balance

### PR DESCRIPTION
**Pull request by gengjun-git**
## What type of PR is this：
- [ ] BugFix
- [ ] Feature
- [x] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Which issues of this PR fixes ：

Fixes #

## Problem Summary(Required) ：

Colocate balance is very time-and-resource-consuming, but when BE is down for a little time, colcoate balance will be triggered. We should avoid colocate balance as much as possible. So add a config param `tablet_sched_colocate_be_down_tolerate_time_s`, if BE is down beyond this time, tablets on that BE of colcoate table will be migrated to other available BEs.
## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

### CODE REVIEW

File: fe/fe-core/src/main/java/com/starrocks/clone/ColocateTableBalancer.java [hunk 1]
```diff
@@ -743,7 +743,7 @@ private boolean checkBackendAvailable(Long backendId, SystemInfoService infoServ
             // 1. BE is dead for a long time
             // 2. BE is under decommission
             if ((!be.isAlive() &&
-                    (currTime - be.getLastUpdateMs()) > Config.tablet_sched_repair_delay_factor_second * 1000 * 2)
+                    (currTime - be.getLastUpdateMs()) > Config.tablet_sched_colocate_be_down_tolerate_time_s * 1000)
```
Comment (nshangyiming) [inline fe/fe-core/src/main/java/com/starrocks/clone/ColocateTableBalancer.java:746]: we shouldn't times 1000 here.

Comment (nshangyiming) [inline fe/fe-core/src/main/java/com/starrocks/clone/ColocateTableBalancer.java:746]: There is test case for this condition in ColocateBalancerTest.java, we need to modify those test cases.

Comment (gengjun-git) [reply fe/fe-core/src/main/java/com/starrocks/clone/ColocateTableBalancer.java:746]: why not, the config param item is second.

Comment (nshangyiming) [reply fe/fe-core/src/main/java/com/starrocks/clone/ColocateTableBalancer.java:746]: yes, you're right

**Review by nshangyiming — APPROVED** [review]

**Review by Astralidea — APPROVED** [review]

**Review by Astralidea — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

**Review by chaoyli — APPROVED** [review]

**Comment by gengjun-git** [comment]
@mergify backport branch-2.5 branch-2.4 branch-2.3 branch-2.2

**Pull request merged**